File: src/main/lab01/Monitor.java
package main.lab01;

// The Monitor Class

class Monitor {

    public String color;
    public String dimension;
    public String resolution;

    Monitor(String color, String dimension, String resolution) {
        this.color = color;
        this.dimension = dimension;
        this.resolution = resolution;
    }

    public void display() {
        System.out.println("Color      : " + this.color);
        System.out.println("Dimension  : " + this.dimension);
        System.out.println("Resolution : " + this.resolution);
    }
}

File: src/main/lab01/Python/Monitor.py
class Monitor:

    def __init__(self, color, dimension, resolution):
        self.color = color
        self.dimension = dimension
        self.resolution = resolution

    def __str__(self):
        return f"""
            Color      : {self.color}
            Dimension  : {self.dimension}
            Resolution : {self.resolution}
        """